src/Estimators/tests/RobotDynamicsEstimator/FrictionTorqueStateDynamicsTest.cpp

```diff
@@ -0,0 +1,328 @@
+/**
+ * @file FrictionTorqueDynamicsTest.cpp
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/generators/catch_generators_all.hpp>
+#include <chrono>
+
+#include <ConfigFolderPath.h>
+#include <iCubModels/iCubModels.h>
+#include <yarp/os/ResourceFinder.h>
+
+#include <iDynTree/KinDynComputations.h>
+#include <iDynTree/Model/FreeFloatingState.h>
+#include <iDynTree/Model/ModelTestUtils.h>
+
+#include <BipedalLocomotion/Math/Constants.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+#include <BipedalLocomotion/ParametersHandler/YarpImplementation.h>
```

Comment:
Looking at the code it seems that you do not use it. I think you can remove it from here and also from the cmake (including the part of the if statement in the CMakeLists.txt)

Reply:
Nope you have to keep the if statement for some of the tests but the tests that do not depend on it you can move them away